src/main/java/uk/gov/register/resources/RecordResource.java

```diff
@@ -118,7 +123,7 @@ public RecordsView records(@QueryParam(IndexSizePagination.INDEX_PARAM) Optional
     }
 
     private IndexSizePagination setUpPagination(@QueryParam(IndexSizePagination.INDEX_PARAM) Optional<IntegerParam> pageIndex, @QueryParam(IndexSizePagination.SIZE_PARAM) Optional<IntegerParam> pageSize) {
-        IndexSizePagination pagination = new IndexSizePagination(pageIndex.map(IntParam::get), pageSize.map(IntParam::get), register.getTotalRecords());
+        IndexSizePagination pagination = new IndexSizePagination(pageIndex.map(IntParam::get), pageSize.map(IntParam::get), register.getTotalDerivationRecords(IndexNames.RECORD));
```

Comment:
Similar to the above. We need to make these two things consistent - do we do the mapping to the correct index function in `PostgresRegister` or here. I know we remove `getTotalRecords()` from the `PostgresRegister` but maybe this could be considered a common function that needs to exist on the `Register` interface.